sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/TestConfigurations.java

```diff
@@ -7,27 +7,33 @@
 
 /**
  * Contains the configurations for tests.
- * 
+ *
  * For running tests, you can pass a customized endpoint configuration in one of the following
  * ways:
  * <ul>
  * <li>-DACCOUNT_KEY="[your-key]" -ACCOUNT_HOST="[your-endpoint]" as JVM
  * command-line option.</li>
  * <li>You can set ACCOUNT_KEY and ACCOUNT_HOST as environment variables.</li>
  * </ul>
- * 
+ *
  * If none of the above is set, emulator endpoint will be used.
  */
 public final class TestConfigurations {
     // REPLACE MASTER_KEY and HOST with values from your Azure Cosmos DB account.
     // The default values are credentials of the local emulator, which are not used in any production environment.
     // <!--[SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine")]-->
     public static String MASTER_KEY =
-            System.getProperty("ACCOUNT_KEY", 
+            System.getProperty("ACCOUNT_KEY",
                     StringUtils.defaultString(Strings.emptyToNull(
                             System.getenv().get("ACCOUNT_KEY")),
```

Comment:
I know this is a testing file but should a different environment variable name be used to reduce chance of collision such as COSMOS_ACCOUNT_KEY?